### PR TITLE
Add E164 Ukrainian phone numbers

### LINKED ISSUE #358: E164 Ukrainian phone number
### Summary

When using `uk_UA` locale given E164 phone numbers (`e164PhoneNumber`) are global.

### Versions

|                | Version |
|:---------------|:--------|
| PHP            | 8.0.3   |
| `fakerphp/faker`  | 1.15.0   |

### Self-enclosed code snippet for reproduction

```php
$faker = Faker\Factory::create('uk_UA');
echo $faker->e164PhoneNumber;
```

### Expected output

Should generate only Ukrainian phone numbers:

```txt
+380508436534
+380668326435
+380682345345
+380964534634
+380673453453
+380913456354
+380921234123
+380934534234
+380946754534
+380956757545
+380964564345
+380974564345
+380984564354
+380634564355
+380994563437
```

### Actual output

```txt
+302534536434
+538453456456
+94567456567
...
```

### PR DESCRIPTION
**Pull request by punyflash**
### What is the reason for this PR?

I was in need to work with a large amount of fake Ukrainian phone numbers and noticed that generated numbers was international. The reason is - E164 formats was not overrided from base phone provider.

- [ ] A new feature
- [x] Fixed an issue (resolve #358)

### Author's checklist

- [x] Follow the [Contribution Guide](https://github.com/FakerPHP/Faker/blob/main/.github/CONTRIBUTING.md)
- [ ] New features and changes are [documented](https://github.com/FakerPHP/fakerphp.github.io)

### Summary of changes

Overrided `$e164Formats` for `Faker\Provider\uk_UA\PhoneNumber` to give only Ukrainian phone numbers.

### Review checklist

- [ ] All checks have passed
- [ ] Changes are approved by maintainer

### CODE REVIEW

**Comment by pimjansen** [comment]
Do we have some source for the formats? Missing these

**Comment by punyflash** [comment]
Source formats described here https://www.itu.int/oth/T02020000DB/en

**Comment by pimjansen** [comment]
> Source formats described here https://www.itu.int/oth/T02020000DB/en

Please add to the code itself so we have a valid ref for later

File: src/Faker/Provider/uk_UA/PhoneNumber.php [hunk 1]
```diff
@@ -48,4 +48,26 @@ class PhoneNumber extends \Faker\Provider\PhoneNumber
         '+38(0###)######',
         '+38(0####)#####',
     ];
+
+    /**
+     * @see Detailed numbering scheme. Source: https://www.itu.int/oth/T02020000DB/en
+     */
+    protected static $e164Formats = [
+
```
Comment (pimjansen) [inline src/Faker/Provider/uk_UA/PhoneNumber.php:56]: Useless whitespace

File: test/Faker/Provider/uk_UA/PhoneNumberTest.php [hunk 2]
```diff
@@ -21,6 +21,17 @@ public function testPhoneNumberFormat()
         );
     }
 
+    public function testE164PhoneNumberFormat()
+    {
+        $pattern = "/^\+?380[1-9]\d{8}$/";
```
Comment (pimjansen) [inline test/Faker/Provider/uk_UA/PhoneNumberTest.php:26]: Please use single quotes instead

**Review by pimjansen — CHANGES_REQUESTED** [review]

**Comment by pimjansen** [comment]
To small things. For the rest it looks good to me

**Review by pimjansen — APPROVED** [review]
Thanks!

**Pull request merged**